nucypher/characters/control/emitters.py

```diff
@@ -46,6 +46,8 @@ def _emit(self,
               message: str = None,
               color: str = None,
               bold: bool = False,
+              request_id: int = -1,  # TODO: What to do when request_id and duration are received?
```

Comment:
I don't really understand the purpose of this, can you elaborate?

Reply:
I can't! As the commit says, it's only a workaround. If you see the signatures of the `_emit` methods in other emitters, there are a `request_id` and `duration` arguments that this method doesn't have and it fails when it receives them. But I don't really don't know what I should with them, hence the comments.

Reply:
haha gotcha